x-pack/plugins/endpoint/common/models/policy_config.ts

```diff
@@ -7,11 +7,9 @@
 import { PolicyConfig, ProtectionModes } from '../types';
 
 /**
- * Generate a new Policy model.
- * NOTE: in the near future, this will likely be removed and an API call to EPM will be used to retrieve
- * the latest from the Endpoint package
+ * Return a new default `PolicyConfig`.
  */
-export const generatePolicy = (): PolicyConfig => {
+export const factory = (): PolicyConfig => {
```

Comment:
removing `generate` from name, as this is a pure function and 'generate' is being used by the 'data generator' scripts and related code